client/modules/i18n/currency.js

```diff
@@ -34,20 +63,19 @@ export function formatPriceString(formatPrice) {
     try {
       // we know the locale, but we don"t know exchange rate. In that case we
       // should return to default shop currency
-      if (typeof locale.currency.rate !== "number") {
+      if (!Match.test(userCurrency.rate, Number)) {
```

Comment:
`Match` also doesn't give you any control over how you log/handle the error.  It just throws a big stack trace.  That's not necessarily bad or wrong, but I personally prefer more customized error handling when it's not inside a server side Meteor method.